package.json

```diff
@@ -2,16 +2,8 @@
   "name": "@near-api-js/monorepo",
   "private": true,
   "engines": {
-    "node": ">=16.14.0"
-  },
-  "workspaces": {
-    "packages": [
-      "packages/*"
-    ],
-    "nohoist": [
```

Comment:
This hoisting is no longer a concern - pnpm doesn't hoist all package dependencies to the workspace root like yarn does, all packages have their own node_modules folder with all their defined dependencies (but these are all symlinked for perf reasons).